cli/src/alluxio.org/cli/cmd/job/submit.go

```diff
@@ -0,0 +1,98 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package job
+
+import (
+	"strconv"
+
+	"github.com/palantir/stacktrace"
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/cmd"
+	"alluxio.org/cli/env"
+	"alluxio.org/log"
+)
+
+var Submit = &SubmitCommand{
+	BaseJavaCommand: &env.BaseJavaCommand{
+		CommandName:   "submit",
+		JavaClassName: cmd.FileSystemShellJavaClass,
+	},
+}
+
+type SubmitCommand struct {
+	*env.BaseJavaCommand
+	operationType string
+	src           string
+	dst           string
+	activeJobs    int
+	batchSize     int
+}
+
+func (c *SubmitCommand) Base() *env.BaseJavaCommand {
+	return c.BaseJavaCommand
+}
+
+func (c *SubmitCommand) ToCommand() *cobra.Command {
+	cmd := c.Base().InitRunJavaClassCmd(&cobra.Command{
+		Use:   Submit.CommandName,
+		Short: "Moves or copies a file or directory in parallel at file level.",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return c.Run(args)
+		},
+	})
+	cmd.Flags().StringVar(&c.operationType, "type", "", "Determine type, options: [cp, mv].")
+	cmd.Flags().StringVar(&c.src, "src", "", "The path to move/copy from.")
+	cmd.Flags().StringVar(&c.dst, "dst", "", "The path to move/copy to.")
+	cmd.Flags().IntVar(&c.activeJobs, "active-jobs", 3000,
+		"Number of active jobs that can run at the same time. Later jobs must wait."+
+			"The default upper limit is 3000.")
+	cmd.Flags().IntVar(&c.batchSize, "batch-size", 1,
+		"Number of files per request. The default batch size is 1.")
```

Comment:
```suggestion
		"Number of files per request")
```
the default value is printed as part of the flag description, so don't need to repeat it